### ISSUE TITLE
CD Rom

### ISSUE THREAD

**Issue by TheBoxGame** [longjunyu2/winlator#31]
add CD Rom support for mounting ISO BIN Discs and more

**Comment by longjunyu2** [comment]
A lot of software can mounting iso etc.. Do we really need it pre installed in winlator?

**Comment by TheBoxGame** [comment]
> Dużo oprogramowania potrafi montować obrazy ISO itp. Czy naprawdę musimy je preinstalować w systemie WinLator?

Yes because I want to mount the iso image of the Medicopter 117 Vol 2 game on a PC but I don't have the right program and a CD-ROM mount player that plays bin and iso images would be useful
